test/e2e/kind.sh

```diff
@@ -30,6 +30,11 @@ nodes:
         nodeRegistration:
           kubeletExtraArgs:
             node-labels: "ingress-ready=true"
+    extraPortMappings:
```

Comment:
Is it really needed?

Reply:
I needed it when running the test on my Fedora, though this is used just for local execution.
Can remove it.

Reply:
Removed for now